compiler/luci/pass/src/ResolveCustomOpBatchMatMulPass.cpp

```diff
@@ -40,9 +40,13 @@ bool resolve_custom_op(luci::CircleCustom *cop)
     batch_matmul->adj_x(map["adj_x"].AsBool());
     batch_matmul->adj_y(map["adj_y"].AsBool());
 
-    replace(cop).with(batch_matmul);
+    auto customOut = loco::succs(cop);
+    assert(customOut.size() == 1);
+    replace(*customOut.begin()).with(batch_matmul);
+
     return true;
   }
+
```

Comment:
This seems unintended format change. @llFreetimell , do you want to revert this?

Reply:
I added this empty line for readability..! Shall I remove this?

Reply:
Usually we separate changes for different purpose :)